app/grandchallenge/cases/models.py

```diff
@@ -292,6 +309,24 @@ class Image(UUIDModel):
     color_space = models.CharField(
         max_length=5, blank=False, choices=COLOR_SPACES
     )
+    patient_id = models.CharField(max_length=64, null=True, blank=True)
+    patient_name = models.CharField(max_length=324, null=True, blank=True)
```

Comment:
Why 324?

Reply:
If I understand correctly this value will consist of 5 components of each 64 characters max, delimited by the `^` character. So that's 5*64 + 4 = 324.

It's described here (PN value representation): http://dicom.nema.org/medical/dicom/current/output/chtml/part05/sect_6.2.html

Reply:
Super! Please add a comment for a future developer there, and again I think this should be checked in the PanImg model so that we can send a nice warning to the user.